subprojects/shipkit/src/main/groovy/org/shipkit/gradle/java/DownloadPreviousPublicationsTask.java

```diff
@@ -14,10 +14,8 @@
  */
 public class DownloadPreviousPublicationsTask extends DefaultTask {
```

Comment:
Can you update the Javadoc of this task? Please try to be as comprehensive as possible!

Reply:
Done